src/pages/Verify/VerifyCodeModal/OtpInput.tsx

```diff
@@ -171,6 +172,8 @@ const OTPInput = ({
       .slice(0, numInputs - activeInput)
       .split('')
 
+    if (isInputNum && pastedData.some(el => isNaN(+el))) return
```

Comment:
```suggestion
    if (isInputNum && pastedData.some(el => isNaN(el))) return
```
this is dont needed

Reply:
no, it is string

Reply:
but typescript err =)) so I will keep it

Reply:
`as any`

Reply:
🙏